### PR TITLE
ADEN-10692 add isMobile targeting parameter

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: platforms/shared/context/targeting/ucp-targeting.setup.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ export class UcpTargetingSetup implements DiProcess {
 			uap: 'none',
 			uap_c: 'none',
 			wpage: wiki.targeting.pageName && wiki.targeting.pageName.toLowerCase(),
+			is_mobile: utils.client.isMobileSkin(this.skin) ? '1' : '0',
```
Comment (nandy-andy) [inline platforms/shared/context/targeting/ucp-targeting.setup.ts:65]: Is there a particular reason for returning strings here and in f2 whereas for sports we're good with numbers (integers)?

Comment (nandy-andy) [reply platforms/shared/context/targeting/ucp-targeting.setup.ts:65]: I think we can make it consistent and return ints everywhere. However, at the end GAM will probably change it to strings, anyway 🤷‍♂️ Yet, we expect to end up with ints ;) https://github.com/Wikia/dfp-creative-templates/pull/129/files#diff-fe8ea114341e4fa86fa2aefba8c026ddb7c5b0677bffccedbf0116f1af3c4c64R184

Comment (jakubbanas) [reply platforms/shared/context/targeting/ucp-targeting.setup.ts:65]: Good point, I changed it to int everywhere 👍

**Review by nandy-andy — APPROVED** [review]

File: platforms/shared/context/targeting/ucp-targeting.setup.ts (hunk 1 above)
Comment (jakubbanas) [reply platforms/shared/context/targeting/ucp-targeting.setup.ts:65]: It turned out that it has to be string, so I made it string everywhere.
When there was int, the template was rendered with an empty string.

**Review by Tomasz-at-Fandom — APPROVED** [review]

**Pull request merged**